github-orgs/kubeflow/org.yaml

```diff
@@ -5,13 +5,16 @@
 orgs:
     kubeflow:
         admins:
+        - andreyvelich
         - caniszczyk
```

Comment:
Should we remoe caniszczyk and krok?

Reply:
Referring to https://github.com/kubeflow/internal-acls/pull/646, they are required to be added by the CNCF.